packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -58,6 +58,117 @@ const {
 const { fakeFileFactory, fakeGranuleFactoryV2 } = require('../../../lib/testUtils');
 const Granule = require('../../../models/granules');
 
+// FUTURE:
+// 1. 'created_at' is updated during PUT/PATCH
+// 2. 'published' defaults to false if not provided in the payload
+// 3. 'duration' comes from the workflow and will be reset on update
+// 4. 'product_volume' comes from a files object on the payload, which may not exist
+//   in the case of partial granule updates
+const cumulusMessageOmitList = [
+  'cumulus_id',
+  'updated_at',
+  'created_at',
+  'published',
+  'timestamp',
+  'duration',
+  'product_volume',
```

Comment:
Should we be validating that the correct product volume is applied based on the files object passed in?

Reply:
We could but I explicitly omitted `product_volume` here because we _are_ testing that we calculate product_volume correctly elsewhere: https://github.com/nasa/cumulus/blob/master/packages/message/tests/test-Granules.js#L158

I think, once fixed, it might be sufficient in this test to remove it from the omit list here and allow it to be included in the comparisons between the original PG granule and the updated one.

Reply:
For clarity, if you submit an update requests it will calculate the product_volume based only on files in the update payload. So e.g. if there are none, the product_volume will be written as zero. We could write a test that proves that issue I suppose.

Reply:
Ah, I see.     This can be resolved in favor of a fix, do we have a ticket?